src/WebExternalSearchProvider.cs

```diff
@@ -251,7 +251,7 @@ private IEntityMetadata CreateMetadata(ExecutionContext context, IExternalSearch
 
         private EntityCode GetOriginEntityCode(IExternalSearchQueryResult<WebResult> resultItem, IExternalSearchRequest request)
         {
-            return new EntityCode(EntityType.Organization, this.GetCodeOrigin(), request.EntityMetaData.OriginEntityCode.Value);
+            return new EntityCode(EntityType.Organization, this.GetCodeOrigin(), request.EntityMetaData.OriginEntityCode.Key);
```

Comment:
I don't think we should provide the full Origin here. What we want to generate here is the OriginEntityCode for this Enricher /EntityType#Origin:UniqueIdValue, which is /EntityType and UniqueIdValue if from the source Entity, Origin is the Enricher